### ISSUE TITLE
Building a sawmill with forestry work table crashes you out of the game

### ISSUE THREAD

**Issue by ryuben** [AlexTheGre4t/MCTNGIssues#31]
http://pastebin.com/7RE5Kp4s

**Comment by AlexTheGre4t** [comment]
Does this recipe involve a TiC axe?

**Comment by Xariyl** [comment]
Script calls for ironAxe in the recipe. ironAxe is variable for a TiC hatchet.

Does this work in a regular workbench?

var ironAxe = TConstruct:hatchet.withTag({
  InfiTool: {
    Head: 2,
    Damage: 0
  }
  });

**Comment by AlexTheGre4t** [comment]
Yes, it does, it most probably is a worktable issue. I'll look into it though

**Comment by Xariyl** [comment]
Since everything else seems to work with the table, I'd agree with mezz that it's likely a TiC/Iguana issue. Need to do some more testing with various mod configurations.

The vanilla tools are still craftable, right? (just not usable in world). In the meantime should we adjust the recipe to use the vanilla tool, or something else entirely?

**Comment by AlexTheGre4t** [comment]
I can't fully remember if they are or not. I'm on vacation now with no PC so I can't check :D

**Comment by ryuben** [comment]
I built one as first thing I did so I assume so

**Comment by ryuben** [comment]
i made the sawmill in a normal crafting table btw - same parts so
